src/test/java/core/FormValidationTest.java

```diff
@@ -76,7 +76,7 @@ private void jsValidation() {
 
     private void navigateAway() {
         jenkins.runThenConfirmAlert(
-                () -> driver.findElement(By.id("jenkins-head-icon")).click());
+                () -> driver.findElement(By.xpath("//a[@href='/']")).click());
```

Comment:
Jenkins may not be running on the root context though.

Reply:
In all these tests it is

Reply:
I wasn't aware of https://github.com/jenkinsci/acceptance-test-harness/blob/master/docs/CONTROLLER.md#existing-jenkins-controller-typeexisting ...